extensions/gitpod-web/src/extension.ts

```diff
@@ -577,6 +578,27 @@ function registerPorts(context: GitpodExtensionContext): void {
 			}
 		});
 	}
+
+	const packageJSON = context.extension.packageJSON;
+	const experiments = new ExperimentalSettings(packageJSON.configcatKey, packageJSON.version, context.logger);
+	context.subscriptions.push(experiments);
+
+	const isSaaSGitpod = context.info.getGitpodHost() === 'https://gitpod.io';
+	async function getPortsViewExperimentEnable(): Promise<boolean> {
+		const isEnabled = isSaaSGitpod
+			? (await experiments.get<boolean>('gitpod.experimental.portsView.enabled', (await context.user).id))!
+			: vscode.workspace.getConfiguration('gitpod').get<boolean>('experimental.portsView.enabled')!;
+		const userOverride = isUserOverrideSetting('gitpod.remote.useLocalApp');
+		context.fireAnalyticsEvent({
```

Comment:
I think we should have rather an event which tells us that user seen a view. This code is executed whenever we create something, like status bar item.

Reply:
@jeanp413 actually maybe let's switch already in VS Code Desktop to sting type as well.